src/storage/src/hummock/sstable/block.rs

```diff
@@ -39,28 +39,28 @@ pub struct Block {
 }
 
 impl Block {
-    pub fn decode(buf: &[u8]) -> HummockResult<Self> {
+    pub fn decode(mut buf: Vec<u8>) -> HummockResult<Self> {
```

Comment:
It will cause twice copy when we try to construct several blocks with one object.

Reply:
For decoding, I think the number of copies is the same before and after this PR.